tests/ui/conftest.py

```diff
@@ -32,35 +46,101 @@ def jwt_issuer(base_url, variables):
 @pytest.fixture(scope='session')
 def jwt_secret(base_url, variables):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
+        hostname = urlparse.urlsplit(base_url).hostname
         return variables['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
-def jwt_token(base_url, jwt_issuer, jwt_secret):
-    payload = {
-        'iss': jwt_issuer,
-        'iat': datetime.datetime.utcnow(),
-        'exp': datetime.datetime.utcnow() + datetime.timedelta(seconds=30)}
-    return jwt.encode(payload, jwt_secret, algorithm='HS256')
+def initial_data(transactional_db):
+    call_command('generate_addons', 10, app='firefox')
 
 
 @pytest.fixture
-def user(base_url, fxa_account, jwt_token):
-    user = {
-        'email': fxa_account.email,
-        'password': fxa_account.password,
-        'username': fxa_account.email.split('@')[0]}
+def create_superuser(
+        transactional_db, live_server, base_url, tmpdir, variables):
+    create_switch('super-create-accounts')
+    call_command('loaddata', 'initial.json')
+
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials=str(tmpdir.join('variables.json')),
+        hostname=urlparse.urlsplit(base_url).hostname
+    )
+
+    with tmpdir.join('variables.json').open() as f:
+        variables.update(json.load(f))
+
+
+@pytest.fixture
+def user(transactional_db, create_superuser, base_url,
```

Comment:
Use the function scoped base URL fixture, otherwise you may have the wrong value.